packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -248,21 +262,29 @@ export const ToastPresentation = React.forwardRef((props, forwardedRef) => {
     hasCloseButton,
     onClose,
     className,
+    contentProps,
     ...rest
   } = props;
 
   const StatusIcon = StatusIconMap[category];
 
   return (
     <Box
+      as='div'
```

Comment:
`as` will come from `...rest`, so does not need to be hardcoded here.

```suggestion
```

Reply:
it's needed to work around typescript issues
https://github.com/iTwin/iTwinUI/pull/1510#discussion_r1307512274

Reply:
The `as='div'` was needed when calling `<ToastPresentation>` (https://github.com/iTwin/iTwinUI/pull/1510/commits/5f48b75c0149125dc1f6f56eb2033c0d054fbc10). But in the code for `ToastPresentation`, the top-most box gets the `as` from the props.

Also, no type errors come when removing L273.

Reply:
Removed